root/otherlookup/form.tt

```diff
@@ -9,7 +9,6 @@
                    , [ 'artist-isni', l('Artist ISNI:') ]
                    , [ 'label-ipi', l('Label IPI:') ]
                    , [ 'label-isni', l('Label ISNI:') ]
-                   , [ 'freedbid', l('FreeDB ID:') ]
```

Comment:
Here is an example: https://musicbrainz.org/otherlookup/freedbid?other-lookup.freedbid=bc0b120d